src/algebra/group/to_additive.lean

```diff
@@ -572,19 +572,16 @@ them has one -/
 { test := (λ d, do
     let mul_name := d.to_name,
     dict ← to_additive.aux_attr.get_cache,
-    match dict.find mul_name with
-    | some add_name := do
-      mul_doc <- doc_string mul_name >> return tt <|> return ff,
-      add_doc <- doc_string add_name >> return tt <|> return ff,
-      match mul_doc, add_doc with
-      | tt, ff := return $ some $ "declaration has a docstring, but its additive version `" ++
-         add_name.to_string ++ "` does not. You might want to pass a string argument to " ++
-         "`to_additive`."
-      | ff, tt := return $ some $ "declaration has no docstring, but its additive version `" ++
-         add_name.to_string ++ "` does. You might want to add a doc string to the declaration."
-      | _, _ := return none
-      end
-    | none := return none
+    add_name ← dict.find mul_name | return none,
```

Comment:
This doesn’t seem to do what I think it does; I still get failure. Undoing that part (and extending the test suite).